src/backend/config/config.json

```diff
@@ -8,8 +8,8 @@
     "host": "localhost",
     "port": 27017,
     "username": "covconnectUser",
-    "password": "covconnect",
+    "password": "2c54c920-7f46-4e73-9e3a-4e78ccb474de",
```

Comment:
hopefully this password is only for your local instance

Reply:
it is, i didn't mean to push this up. i'll undo these changes